manifests/kustomize/base/cache/cache-deployment.yaml

```diff
@@ -18,6 +18,11 @@ spec:
       - name: server
         image: gcr.io/ml-pipeline/cache-server:dummy
         env:
+          - name: CACHE_IMAGE
+            valueFrom:
+              configMapKeyRef:
+                name: pipeline-install-config
```

Comment:
I believe you also need to update pipeline-install-config generated by the base/params.env